lib/ansible/modules/system/group.py

```diff
@@ -77,7 +77,8 @@
 import grp
 
 from ansible.module_utils._text import to_bytes
-from ansible.module_utils.basic import AnsibleModule, load_platform_subclass
+from ansible.module_utils.common.sys_info import get_platform_subclass
+from ansible.module_utils.basic import AnsibleModule
```

Comment:
Alphabetize imports.
```suggestion
from ansible.module_utils.basic import AnsibleModule
from ansible.module_utils.common.sys_info import get_platform_subclass
```